app/styles/less/experiments.less

```diff
@@ -354,3 +354,22 @@
     text-align: center;
   }
 }
+
+.training-and-validation {
+  display: flex;
+  flex-wrap: wrap;
+  &>div {
+    margin-right: 1em;
+    &:last-of-type {
+      margin-right: 0;
+    }
+    &>div {
+      border: 1px solid #9d9d9d;
+      .datasets-types {
```

Comment:
in variables.less there is some sass vars that you can use. Maybe @gray-light is ok here

Reply:
thanks